tamr_client/_types/backup.py

```diff
@@ -0,0 +1,24 @@
+from dataclasses import dataclass
+
+from tamr_client._types.url import URL
+
+
+@dataclass(frozen=True)
+class Backup:
```

Comment:
Do we need a better name since `Backup` here refers to the operation, not the resulting .zip artifact?

Reply:
Tamr refers to these (the JSON objects that contain information about a backup operation) as backups so i think its fine to leave as is

Reply:
Can you add to the docstring saying that?